apps/sms/test/marionette/lib/views/inbox/view.js

```diff
@@ -17,4 +17,12 @@ InboxView.prototype.goToFirstThread = function() {
   return conversation;
 };
 
+InboxView.prototype.createNewMessage = function() {
+  this.accessors.createNewMessageButton.tap();
+  var ComposerView = require('../composer/view');
+  var composer = new ComposerView(this.client);
+  composer.accessors.waitToAppear();
```

Comment:
Putting waitToAppear feels like the method can be used in the test. I'm not sure that's what we want, so that's why I put it in the accessors. I propose we leave it the current way, and see how get the new PRs. Maybe it's cleaner to have this kind of method in the view.